### PR TITLE
Changed migration files according to new db design

### PR DESCRIPTION
**Pull request by saramcallister**
The python scripts aren't updated so this may wreak havoc on the develop branch.

### CODE REVIEW

File: teamworkApp/db/schema.rb [hunk 1]
```diff
@@ -20,10 +20,43 @@
...
+    t.integer "communicator"
+    t.integer "collaborator"
+    t.integer "challenger"
+    t.integer "contributor"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "teams", force: :cascade do |t|
+    t.string "name"
```
Comment (hmc-cs-mcmurphy) [inline teamworkApp/db/schema.rb:59]: How does this connect to the data about which students are in the teams? I see there is a string `name` in the teams table, but an integer `team_id` in the students table. I could just be confused with the structure of databases and how these connect. I could also be missing another detail...

**Review by hmc-cs-mcmurphy — CHANGES_REQUESTED** [review]
Looks good! I just have one question about the teams and students tables.

**Comment by hmc-cs-mcmurphy** [comment]
Looks good to me and it seems to capture our discussion.
I don't actually request changes I just wanted the option that would mean I'd like clarification on the tables before we merge and I pull. 
Nice job, thanks Sara!

File: teamworkApp/db/schema.rb (hunk 1 above)
Comment (saramcallister) [reply teamworkApp/db/schema.rb:59]: So teams have many students, which means to represent teams without requiring a fixed size of team, we need the students to each have an attribute which identifies the team they are on. The team_id in students allows this to happen.

Comment (hmc-cs-mcmurphy) [reply teamworkApp/db/schema.rb:59]: Ok, so how does this connect back to the teams table? Like, how could you know the name of their team?

**Review by hmc-cs-mcmurphy — APPROVED** [review]

**Pull request merged**